homework/generate-sequence/generateSequence.hpp

```diff
@@ -2,6 +2,13 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
+    std::vector<int> vec;
+    if (count < 0) count = 0; // return empty vector for negative count
```

Comment:
just return here. And move this to the top (before create vec). 
One line one command :)